docs/transaction.md

```diff
@@ -175,5 +175,11 @@ console.log(transaction.getLockTime());
 // output similar to: Sun Nov 30 2025 00:00:00 GMT-0300 (ART)
 ```
 
+## Simple Transaction
+
+Since 12.4, simple transactions (having no more than 4 inputs), are entitled to an auto Instant Send upgrade.  
```

Comment:
v13? 😉

Reply:
Maybe it is worth mentioning that simple transactions are

> entitled to an auto Instant Send upgrade. 

only as long as the mempool is less than 10% full.

https://github.com/dashpay/dash/blob/master/src/instantx.h#L50-L52
https://github.com/dashpay/dash/blob/master/src/instantx.cpp#L40

Reply:
I actually double checked, it's on milestone 12.4 that it has been shipped :3 
Should I change to v13 instead ? 
Good suggestion Cofresi, will apply in same time than v13 if the change is needed :)

Reply:
@Cofresi : Seems good like that ?

Reply:
Perfect!

Reply:
@Cofresi @nmarley I still need a approval tho :P

Reply:
Sorry, didn't know we had a 12.4 milestone. I think it shipped w/v13 but not too important a detail anyway.